external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -1623,7 +1622,7 @@ impl Symbolicator {
                         &tname,
                         ident_type,
                         ident_type_def,
-                        doc_string,
+                        "".to_string(), // no doc string for type params
```

Comment:
Probably more elegant to represent this with `None`, but I don't know how invasive that change becomes. If it's super easy, maybe worth it?

Reply:
Good point! My Rust mojo was even worse then than it is now :-) It's good time to clean it up!